platform/barefoot/docker-syncd-bfn-rpc/Dockerfile.j2

```diff
@@ -27,6 +27,8 @@ RUN apt-get update \
     python-dev          \
     wget                \
     cmake               \
+    sensord             \
+    fancontrol          \
```

Comment:
sensord and fancontrol are already in pmon docker, why do we need them in syncd docker?

Reply:
For bfn platforms, sensors information is not available on the host cpu and is fetched from the BMC. The infra needed to retrieve this information is available present in the syncd container, which we leverage for retrieving needed information from the BMC. Hence the need to have these packages in syncd

Reply:
can you pass the information from syncd to the pmon docker? we'd like to unify the those controller in one place for all platforms?

Reply:
That would need us to install all python libraries on pmon container along with some rpc infra to communicate with syncd container.  I do not know of any other clean mechanism to pass data between two containers. Hence the need to have these packages on syncd itself